stix_shifter_utils/stix_translation/src/json_to_stix/json_to_stix_translator.py

```diff
@@ -298,6 +298,14 @@ def _handle_value(self, data, parent_data, ds_sub_key, to_stix_config_prop, obje
 
                 transformer = self.transformers[prop['transformer']] if 'transformer' in prop else None
                 references = references = prop['references'] if 'references' in prop else None
+                
+                # adding object index in references
```

Comment:
can you put more details here on what type of fields it is used for translation? maybe a small example since its going into the common code.

Reply:
example and comment is hard to understand. is there any usecase in the current pr? which mapping attribute in log analytics uses this reference feature? any example result?

Reply:
explanation for this case scenario is sent in mail, please check it.

Reply:
Comments were added as per your suggestion